s4e-backend/src/main/java/pl/cyfronet/s4e/ex/ErrorHandler.java

```diff
@@ -85,6 +85,11 @@
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorHandlerHelper.toResponseMap(e));
     }
 
+    @ExceptionHandler(GroupNameException.class)
+    public ResponseEntity<?> handleGroupNameException(GroupNameException e) {
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorHandlerHelper.toResponseMap(e));
```

Comment:
In this way the response will only contain the __general__ info. Maybe create a custom toResponseMap method, or validate with a custom validation annotation? (e.g., https://grokonez.com/spring-framework/create-custom-validation-spring)